lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -32,6 +32,7 @@ <h4 class="list-group-item-heading">
             </span>
 
           </footer>
+          <progress ng-if="drive.progress || true" value="{{ drive.progress || 50 }}" max="100"></progress>
```

Comment:
Would it make more sense to not display the progress bar at all if the drive has no progress?

Reply:
Yup, my bad! The conditionals are only there for testing, I don't remember why I decided to keep them when I pushed...